styles/index.css

```diff
@@ -0,0 +1,26 @@
+header {text-align: center;}
+
+ul {
+    list-style-type: none;
+}
+section.info {
```

Comment:
There should be a white space between a closing bracket and the start of the next rule